app/src/main/res/layout/activity_movie_details.xml

```diff
@@ -0,0 +1,241 @@
+            android:contentDescription="@string/star_1" />
+
+        <!-- Star Icon -->
+        <ImageView
+            android:id="@+id/star_icon_2"
+            android:layout_width="@dimen/amd_star_width"
+            android:layout_height="@dimen/amd_star_height"
+            android:src="@drawable/ic_star_24"
+            app:layout_constraintBottom_toBottomOf="@+id/star_icon_1"
+            app:layout_constraintStart_toEndOf="@+id/star_icon_1"
+            app:layout_constraintTop_toTopOf="@+id/star_icon_1"
+            app:tint="@color/selected"
+            android:contentDescription="@string/star_2" />
+
+        <!-- Star Icon -->
+        <ImageView
+            android:id="@+id/star_icon_3"
+            android:layout_width="@dimen/amd_star_width"
+            android:layout_height="@dimen/amd_star_height"
+            android:src="@drawable/ic_star_24"
+            app:layout_constraintBottom_toBottomOf="@+id/star_icon_2"
+            app:layout_constraintStart_toEndOf="@+id/star_icon_2"
+            app:layout_constraintTop_toTopOf="@+id/star_icon_2"
+            app:tint="@color/selected"
+            android:contentDescription="@string/star_3" />
+
+        <!-- Star Icon -->
+        <ImageView
+            android:id="@+id/star_icon_4"
+            android:layout_width="@dimen/amd_star_width"
+            android:layout_height="@dimen/amd_star_height"
+            android:src="@drawable/ic_star_24"
+            app:layout_constraintBottom_toBottomOf="@+id/star_icon_3"
+            app:layout_constraintStart_toEndOf="@+id/star_icon_3"
+            app:layout_constraintTop_toTopOf="@+id/star_icon_3"
+            app:tint="@color/selected"
+            android:contentDescription="@string/star_4" />
+
+        <!-- Star Icon -->
+        <ImageView
+            android:id="@+id/star_icon_5"
+            android:layout_width="@dimen/amd_star_width"
+            android:layout_height="@dimen/amd_star_height"
+            android:src="@drawable/ic_star_24"
+            app:layout_constraintBottom_toBottomOf="@+id/star_icon_4"
+            app:layout_constraintStart_toEndOf="@+id/star_icon_4"
+            app:layout_constraintTop_toTopOf="@+id/star_icon_4"
+            app:tint="@color/no_selected"
+            android:contentDescription="@string/star_5" />
+
+        <!-- Reviews -->
+        <TextView
+            android:id="@+id/reviews"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="@dimen/spacing_small_8"
+            android:gravity="center_vertical"
+            android:lineSpacingExtra="4sp"
+            android:text="@string/reviews"
+            android:textAppearance="@style/Reviews"
+            app:layout_constraintBottom_toBottomOf="@+id/star_icon_5"
+            app:layout_constraintStart_toEndOf="@+id/star_icon_5"
+            app:layout_constraintTop_toTopOf="@+id/star_icon_5" />
+
+        <!-- Name -->
```

Comment:
This view isn't a name, right? So let's just remove this comments. What do you think?

Reply:
Deleted.